js/src/common/components/Modal.tsx

```diff
@@ -129,7 +129,9 @@ export default abstract class Modal<ModalAttrs = {}> extends Component<ModalAttr
   /**
    * Handle the modal form's submit event.
    */
-  abstract onsubmit(e: Event): void;
+  onsubmit(e: Event): void {
```

Comment:
...while we're here! 😛

```suggestion
  onsubmit(e: SubmitEvent): void {
```